wasmsdk/blobber.go

```diff
@@ -493,7 +495,7 @@ func uploadWithJsFuncs(allocationID, remotePath string, readChunkFuncName string
 		numBlocks = 100
 	}
 
-	ChunkedUpload, err := sdk.CreateChunkedUpload("/", allocationObj, fileMeta, fileReader, isUpdate, isRepair,
+	ChunkedUpload, err := sdk.CreateChunkedUpload("/", allocationObj, fileMeta, fileReader, isUpdate, isRepair, webStreaming
```

Comment:
I would suggest you to use `WithWebStreaming(status bool)` instead of `webStreaming` input. It is safer to other apps.

Reply:
Won't be able to add it as an option since we would like web streaming to change properties (file reader and file meta ) before su creation. https://github.com/0chain/gosdk/pull/905/files#diff-58fd13504400641409e5eaff205730ab3b97d39e35d81e9a39cd981d2a05a930L101

Reply:
ok